src/commands/flinkStatements.ts

```diff
@@ -24,8 +43,143 @@ export async function viewStatementSqlCommand(statement: FlinkStatement): Promis
   await vscode.window.showTextDocument(doc, { preview: false });
 }
 
+/**
+  * Submit a Flink statement to a Flink cluster. Flow:
+  * Quickpick flow:
+	*  1) Chose flinksql, sql, or text document, preferring the current foreground editor.
+  *  2) Statement name (auto-generated from template pattern, but user can override)
+	  2) the Flink cluster to send to
+    3) also need to know at least the 'current database' (cluster name) to submit along with the catalog name (the env, infer-able from the chosen Flink cluster).
+  5) Submit!
+  6) Raise error box if any immediate submission errors.
+  7) Refresh the statements view if the view is set to include the cluster
+*/
+export async function submitFlinkStatementCommand(): Promise<void> {
```

Comment:
Ahh I see - follow-up branch to add `item?: CCloudFlinkComputePool` as an arg here and allow attaching the command to individual pools (like we do for SR)? Could then fall back on the quickpick if nothing is passed.